src/main/webapp/partials/instructorHelpCourses.jsp

```diff
@@ -821,6 +833,282 @@
       </div>
     </div>
   </div>
+  <h3>Restoring Courses</h3>
```

Comment:
Should the subsection be "Deleting Courses" instead? It seems like the bulk of this section concerns deleting courses (beginning from the very first collapsed panel). 

Besides, the preceding section is also titled "Archiving Courses" and not "Unarchiving Courses" :P

Reply:
I'm considering whether it is a bit confusing to users since most of the times "deleting courses" refers to deleting from existing courses table (soft-deletion in this case). Restoring courses should be the major feature of Recycle Bin, same as archiving courses.

If "Restoring Courses" is really not suitable, what about "Deleting Courses From Recycle Bin" instead?